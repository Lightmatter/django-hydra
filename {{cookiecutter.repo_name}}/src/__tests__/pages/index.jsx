import React from 'react';
import Index from '{{cookiecutter.repo_name}}/src/pages/index';
import { mount } from 'enzyme';

it('renders homepage unchanged', () => {
  const tree = mount(<Index />);
  expect(tree).toMatchSnapshot();
});
