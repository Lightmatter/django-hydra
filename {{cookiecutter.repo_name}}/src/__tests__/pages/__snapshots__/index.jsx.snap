// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<Index>
  <h1>
     Hello World 
  </h1>
</Index>
`;
